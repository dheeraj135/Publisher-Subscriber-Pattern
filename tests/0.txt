S topic5 2
U topic9 2
U topic0 2
T topic6 584aBiOzfniRzgR
S topic8 0
S topic7 4
U topic6 1
S topic8 4
S topic1 0
S topic7 4
U topic5 1
S topic5 3
U topic0 4
U topic5 0
T topic4 LGebAMNuyM6ao8v
T topic5 txZj5JJp3KCYqO9
T topic9 bQe3BLAORN3nHE2
S topic5 3
U topic3 0
T topic9 1Bl831oDfNhuqag
U topic0 2
S topic4 4
T topic9 9zWUzZBDVkVCmMr
U topic7 4
T topic0 FHJJ2goFzcny8dk
U topic1 2
U topic9 0
T topic1 30nq30KMHvOS6kd
T topic8 1G7JH7U75PlMod7
U topic9 3
U topic3 3
T topic4 ghgigUrTgNQvFNS
U topic2 1
S topic9 2
U topic7 1
U topic2 4
S topic3 4
S topic1 0
S topic5 3
U topic0 3
S topic3 4
U topic7 3
S topic0 0
U topic8 0
T topic5 iBqAUByuxib3Px2
U topic7 0
T topic4 ZxXmY9XOWVn8BOH
T topic9 m3lq7NrITigR5CO
U topic0 2
U topic6 0
U topic0 4
U topic8 2
U topic6 1
T topic7 C9jLPTZ6n6e4IyM
T topic6 Hz0WNuHevuyeOqU
S topic9 4
T topic6 MlsDHL6JmF0KYg4
T topic5 xaIqTJqvcfdDOYf
T topic2 Lm8CvELQBU1SEy5
T topic2 1uMBb9Li4ukQiE5
T topic1 9aiMi2TZxGyoLkI
T topic4 ut9ELbYJU02yTnJ
S topic9 1
T topic5 EOEaghNBHN3s0gs
U topic3 4
U topic7 0
S topic9 3
T topic1 hzcUThGTl7yZ7yt
T topic7 nprt4tKrbsct8Bh
T topic2 Wy5fF1geCgj9jxE
S topic2 2
U topic5 4
S topic4 4
U topic6 4
U topic1 4
U topic1 2
S topic6 2
U topic1 1
U topic7 1
T topic1 quBhP4vYdVxOBTr
S topic7 2
S topic7 0
S topic1 0
U topic5 1
T topic1 fMOGNogNxiQt1L8
S topic7 2
T topic8 LVvlmy1EtjdzYWK
S topic8 4
U topic7 2
U topic7 3
T topic3 KQuU6L9lX97JCGu
T topic2 xbBRaPddBegrhFH
T topic0 DiiSupUaHEPPDGT
T topic8 mFkgXNGUIZqSjcc
S topic8 2
T topic9 r4J9hD5Uz9cBLq9
S topic6 0
T topic5 6Cluom8GlFs8543
T topic9 koV1YU4NpAcBjkv
U topic4 1
S topic3 0
U topic8 1
U topic9 2
T topic5 YSR4Cztlvo0tDUg
T topic9 lTFEOJ9Q7W3czYv
U topic4 2
T topic2 RQhRUuJGn9uuuxi
T topic3 0D8ov4PspXE7cr9
U topic3 2
U topic0 4
U topic7 2
T topic0 ybx84731gWE0KYG
T topic5 qyoTUSphhQQmuEb
U topic1 2
T topic3 ouT7I1VYCNGVAZu
S topic8 0
U topic2 2
S topic2 1
U topic0 4
T topic3 pvilm2r36TM9kSr
T topic9 9ffNv9GoIzClf8s
U topic8 1
U topic7 0
S topic7 0
S topic7 3
S topic3 4
T topic6 QlrqR1KWC6B9eZK
U topic5 2
S topic4 3
U topic9 4
U topic1 0
S topic0 3
S topic6 2
U topic1 2
S topic0 1
U topic0 0
T topic7 PvXvlGW0PKdH2jp
S topic7 1
S topic2 4
U topic3 3
U topic2 1
T topic9 O21bi4VOt75VJUa
S topic2 4
U topic3 1
S topic8 2
S topic7 2
T topic8 g7hEugvkPz2t2Sr
U topic8 1
T topic8 hgMc1iG09FrKvzN
T topic1 TgdpiJisrSrbnPF
T topic5 GLUnM4SSX1Xnr1T
S topic4 1
U topic7 4
U topic6 2
T topic5 1x13XgxGTrAQsr4
T topic8 JQinmMT80FvsLXv
S topic2 0
T topic4 Fbeie007bUe234R
S topic5 3
U topic9 0
T topic5 Xv6fmv0cfLdWkPm
U topic8 0
U topic5 3
U topic7 0
S topic8 2
S topic0 0
T topic3 8qH8jtmLgAZ0cFZ
U topic2 1
S topic5 4
T topic3 Q5CSljYG8K4W6DX
U topic2 0
U topic7 3
T topic9 ILhvxs2dmF9ZHDo
U topic8 0
S topic1 3
U topic9 3
T topic7 YY5GXD2m2BkxPfs
U topic9 1
T topic4 OAU2rId8EnLy0ix
T topic6 JgyQ4QRyYZThHKH
S topic5 4
U topic6 0
S topic6 1
S topic7 1
T topic1 7ekoOBKM34KGkKJ
T topic4 9cVkSKbqoRnH0iA
T topic8 YnfpPj5zTWhVF96
S topic5 0
U topic1 0
S topic9 3
T topic1 kGgTfPHh2g4FoJa
U topic3 4
T topic4 oqHZquOItD6zehQ
T topic4 TtGTXGZqo9NubyS
S topic8 2
S topic3 0
S topic6 3
T topic7 bKh02iqUaI1P8gz
U topic3 0
U topic7 1
S topic1 0
T topic4 WHUMeo57WqQPKjs
U topic1 2
U topic1 1
T topic3 AetdKFRegTQpnUG
T topic8 aNAtnQvU44NUlA8
S topic7 4
T topic2 14sRR4X8YAJuKFy
U topic0 2
T topic1 FSs7CF73lKiUNmX
S topic3 4
U topic0 3
T topic0 Ni7FWukO4iJW2VW
T topic9 QEJf9silbuuWFKJ
U topic2 2
T topic3 zVqVSQrZNiZqKZU
T topic3 46Z8vJ7N72QsjJt
T topic7 63vtNDYR2klkuuW
S topic1 2
T topic1 XFjbgbn0pAkDQ5P
T topic5 4yPjPPxUvrAP0pA
U topic5 0
U topic8 4
U topic9 2
S topic0 3
T topic5 SSgPAmuh87MGVFM
U topic8 3
U topic7 2
U topic4 4
T topic9 i0Gs2ThnX3c3bDs
U topic8 0
T topic6 SYfaEQCb79JlpjG
S topic5 3
T topic1 PL5e5jGPCmvFTaN
S topic2 2
U topic6 3
T topic9 uk9ZIndXB4fUbd5
T topic3 qxIrTa8sX5ypx19
S topic9 0
U topic0 2
U topic1 2
U topic5 3
U topic1 2
U topic9 4
S topic4 3
U topic2 2
S topic2 4
S topic1 3
T topic6 4bIqoDa04BSW5lR
U topic1 0
T topic5 VjbdLcjFvBiWaIh
T topic7 9nJ7oogR1IVH2aB
U topic1 3
U topic9 3
S topic3 0
T topic0 LTQ0L6Ar77LcpEs
T topic6 baIXSJYGXd2V7Bg
U topic9 2
T topic0 MiBRq7pjnxkXo4G
U topic2 3
T topic7 o51lrHycRsAEuAQ
U topic9 1
S topic8 0
S topic9 2
S topic2 1
U topic2 2
T topic4 QOQtPLkRL6NtNmg
T topic3 9HttnmJACJxfIvy
T topic3 9DBPdtaiCOmrNkM
U topic2 4
S topic6 4
S topic0 2
S topic3 1
T topic9 qhIeH8vhlKNEIkd
S topic1 3
S topic6 1
T topic8 1VW1XrrqNctBpXX
T topic3 y0hhCe2bURHDMQS
S topic2 4
U topic3 2
U topic6 1
S topic1 0
T topic9 KgIPACP2X4b23AW
S topic4 1
T topic7 V0DPF9WeFcN71yC
T topic1 s1iJj1QdNNsSj8s
S topic7 4
T topic9 VUKy9DJiRXZeY41
U topic4 0
S topic3 3
S topic6 3
T topic5 ipMA6FQgGnovHr1
T topic8 NbTXu1VB1eKbd9N
T topic9 KHdfpJ2eTYo4GEj
T topic1 9CJGn2PYpDKypNT
T topic7 cmsqihKRHEb3mHL
S topic2 0
T topic8 C41KO9IB2zsylZR
S topic6 1
U topic8 2
T topic4 dVN7jkLdvitCv7C
S topic9 3
T topic1 fYnjOF0JEJtjdyd
S topic4 0
U topic0 0
S topic7 0
T topic7 V2YLBxYKHWQk1MF
S topic5 0
U topic3 3
S topic2 0
S topic3 1
S topic5 0
S topic5 4
S topic4 0
T topic6 RYU0bBqszJEN96l
U topic6 3
T topic2 SIXWzLM7Ec3rjnL
U topic9 1
T topic1 EPEkDqFCCOxhvgB
S topic0 2
U topic0 3
S topic0 4
T topic8 opQtS0vSEKooSXt
T topic7 BOAcu8am7tHLmb3
U topic6 0
T topic8 EWKGWVKGRDq2DDJ
T topic7 qrs69KaanA53Q00
T topic3 ZcSjyVm1lfAgjfR
S topic4 2
S topic8 1
U topic0 4
S topic6 0
U topic2 0
S topic8 0
S topic9 3
S topic9 4
T topic5 er4PVLIgBEyB5Fk
T topic7 81hfNu4KFiPZDaL
S topic4 4
U topic9 1
S topic6 4
S topic8 3
U topic0 2
U topic8 2
T topic3 cGA5pUxJbNtneeN
U topic5 4
S topic7 1
S topic4 2
U topic1 0
U topic5 0
T topic0 4kl2mq0ICVI8DrU
S topic0 2
S topic4 2
S topic2 2
S topic2 4
S topic7 3
T topic3 x6t637j6WB87epa
T topic2 kaCAkv3GSJ79Ufz
S topic4 1
T topic2 8Hij73oXvyHL7D8
T topic8 KyLbEIoFKDOmPZD
U topic4 1
S topic0 4
U topic8 1
T topic8 xigcm382kCb5bPe
S topic1 3
T topic5 kDmqAugtQEPZSUh
U topic1 4
S topic9 2
U topic3 0
T topic6 0WZZWvV3JjmHVpU
U topic4 4
S topic9 1
U topic7 4
T topic7 9Ui7yuByavC5a1u
U topic0 3
T topic7 7Fgt49rzgBn2SBP
T topic4 uUN18Km43TW5cI4
T topic5 iAj92L9QTEAksMq
S topic0 0
S topic9 3
S topic5 1
S topic6 3
T topic5 0Yezz9PyqfCHmlL
S topic2 4
U topic1 3
T topic1 lhOMIPEzl30SeqY
S topic6 1
S topic5 4
S topic5 3
S topic4 0
T topic7 mpz6nAseQW1qCBe
T topic4 Cu7keZ1vLCsmh96
T topic5 LuOOTqJcfiqgh3v
S topic3 2
U topic1 2
U topic3 1
U topic2 3
T topic5 Gc3jUcoen7PAmRl
T topic5 ckxtDkKUvHnzfJM
U topic5 1
S topic2 0
T topic3 yFQYzI35bEo9Hhp
T topic3 gsYd5NjJyx564M5
S topic9 2
U topic3 0
U topic8 4
S topic8 2
S topic1 3
T topic2 xduflNRSlvrMymC
T topic8 3eludqXn8j17QgD
U topic0 2
S topic2 4
S topic9 4
S topic6 2
U topic4 2
U topic4 4
U topic6 0
S topic2 2
T topic4 K3iCgyB2U9cIp8P
U topic5 3
U topic2 2
T topic1 RpzQCUmlam7Xeam
U topic3 1
S topic1 1
T topic6 1eaPHpzp9gxA476
U topic9 3
S topic9 3
S topic5 3
T topic3 0xfWFM8aCONdkYO
T topic9 YSECJgh6v06jgRh
S topic0 3
U topic7 4
U topic8 2
T topic2 ldrpgMFJLlh4Oln
S topic3 3
T topic8 Ah5RKlmdM1QIDMy
T topic6 rR87E5l7JcvXsS4
T topic2 MbsZ0344SBkgluG
S topic5 2
S topic6 3
S topic2 2
T topic9 D1ZPP6mzN3BHd3O
T topic1 XJdZBT0yRfJIe7G
S topic6 4
U topic0 0
T topic6 fbgdTCPZTbUXyMq
T topic7 yGHFfCgQnejW5RP
U topic3 2
T topic0 scFeBrgLzSVGy8A
T topic3 LASx5pVYy4NskXb
T topic4 5yCk3HvT1VubWzP
S topic2 4
S topic6 0
S topic6 1
U topic2 2
T topic2 TSqlMpPaoRKSY0r
S topic1 3
T topic5 fUntiXK0zYj4uUK
T topic9 NaBs1oqYO52UgAB
U topic9 0
T topic1 8avxRR38pcU38Sp
U topic6 3
T topic5 efuxdfubPqORDVN
U topic4 1
T topic1 Nu9n3Or3A8Wetfb
S topic7 2
T topic2 tWcxeSV7aai1kxU
T topic7 jrN1Y8WnEGFBivs
T topic1 z3uHT4iV9sibqxU
U topic4 0
S topic9 1
T topic7 ygYymJ4AldEZ94Q
S topic6 3
U topic3 3
S topic9 2
U topic3 1
S topic2 3
U topic4 4
S topic8 3
U topic5 1
T topic2 KTiB7xBnjcFRi88
S topic6 2
U topic1 1
T topic7 ep3P7jEgFiavCxb
T topic5 FMUFHzDcmK9d4YQ
T topic2 SYXNvEJ2nB6IVKj
U topic5 1
U topic4 0
T topic0 2Q3RDIKu77RgX1v
U topic2 1
S topic7 2
U topic6 0
S topic2 3
T topic4 PtiOoZpcWlrkBQQ
S topic7 2
T topic2 WTANLpKpoQpukbD
U topic4 3
U topic1 3
S topic0 2
T topic5 UBcpTDO7pfTPvp3
S topic3 1
U topic1 2
S topic4 1
U topic5 3
U topic2 4
S topic2 2
U topic2 2
S topic9 2
T topic6 iiiWYnuzZgTmjGW
S topic9 3
T topic7 ojqBpsdDNREHDPX
T topic7 5ixm1UlX1GaSpO3
S topic1 3
S topic0 4
U topic0 4
T topic6 HfhoEQ7lYBrdcVu
U topic2 0
U topic4 0
S topic5 4
U topic5 3
U topic9 3
U topic5 3
U topic9 0
U topic1 1
U topic5 4
T topic9 CcXtlHjz9h29q5X
S topic4 0
U topic0 0
S topic3 3
T topic7 IWb4GvCuFf0rGk2
T topic9 kAA6uLzHKTgqrmL
T topic1 rqBoopfV9sszPaS
U topic0 0
U topic3 0
T topic6 LX0rZGFb4OOajKN
U topic3 0
S topic9 3
T topic5 RRS8l9BT0cOxIRb
T topic4 QHR0LbD2NU3gJdr
S topic4 1
T topic8 71X8DfTFC8628Si
S topic6 1
U topic6 4
T topic7 h1YbrUqJ2GOAtPB
U topic0 1
S topic6 3
U topic8 0
U topic4 2
U topic4 3
T topic3 57Apj4pQYRHGPk9
T topic2 M5VBcdK1jfMUz0P
U topic5 2
S topic9 0
U topic1 2
T topic2 YxOhZS4HWUo3CUZ
U topic1 4
S topic0 2
T topic4 D9qiIT3XzfZ00zA
U topic4 1
S topic9 3
T topic5 54hrtO6jYrsOndZ
S topic5 0
S topic8 2
T topic7 qpHDjfkj1PJtfjk
T topic3 VV0YZHB4UtTbPSb
S topic9 2
S topic0 4
U topic8 2
S topic6 2
S topic7 4
U topic1 1
T topic7 pTyHmBvhCv2BK5o
T topic5 SV2kAtXfDV3NxyC
T topic4 h2qG5kFZqWWCWHI
S topic4 3
T topic8 GNcL6I2zIXRLHKs
T topic4 pAccT3nWduKt4xn
T topic5 RziupjcHN8R7pYu
T topic9 cjPi9ohsEROzBx9
U topic8 4
T topic8 JVXnDohjhEKjFlG
T topic2 xTefTbFuBGnSgz4
U topic1 4
S topic5 3
T topic4 312n8cynrQmJDU0
U topic9 0
T topic3 BJXH4IfC6kSAjkz
T topic0 kPxFTqaQ0TXH0oJ
T topic2 aosirKipukUQhji
U topic3 0
S topic2 3
U topic3 4
U topic1 1
S topic4 0
T topic5 pdV3r1ErZ92C2VZ
T topic2 3KLrOOc4kz2ePve
S topic0 1
S topic8 3
S topic4 4
S topic8 3
T topic3 BIESEIW1eCGl6y5
U topic4 3
T topic9 ImTX2ouCb4bi6dF
U topic3 4
T topic0 XWjTI9bTNQ9dqNA
S topic8 0
T topic0 gUpA5Kr6AJT5pv9
T topic7 o5R1mYxQgBDoAm2
T topic3 QFliW0h8o0suCai
S topic5 2
U topic1 4
U topic2 1
S topic4 3
S topic9 1
T topic1 MgvO3vKXuafCl9H
T topic7 Io6ldMhC8N7GDLK
T topic5 8pz7GWV6VV4seLP
T topic1 7sBpLO4voXh90ij
U topic3 0
S topic2 4
T topic1 cuNfJSfC7xlDy0e
U topic2 0
T topic4 hR9ss8jTcMfpxS0
U topic6 4
U topic8 2
T topic4 CIOIugiC1Tijoe6
S topic3 2
T topic0 9Kzpq1j2xYsgaBd
U topic5 3
S topic0 0
U topic4 3
S topic0 0
S topic4 1
U topic2 3
S topic2 3
S topic6 4
T topic5 ss1aQ7W3nMIRpKn
S topic7 3
U topic3 3
T topic5 q4Hp51fHJFxZ2Xq
U topic6 1
U topic3 2
T topic0 6Se3WOMCbTq5z10
U topic1 3
U topic2 4
T topic1 2VDxu6Cs4POulMp
T topic2 64uctZrmJZSkKum
T topic6 XSQyv4i5zOJzD4r
U topic0 3
S topic7 1
U topic5 3
U topic9 0
U topic3 2
S topic8 2
S topic6 2
U topic3 1
U topic2 4
U topic6 2
U topic0 3
T topic4 CIyZs7QCbcZMRl8
U topic8 1
U topic4 0
U topic4 3
U topic6 1
T topic0 vAaWTxVQTBRxZTN
T topic2 gNY56ilh2ZnHPpU
U topic8 1
U topic0 0
S topic4 3
T topic3 o1ONbq1WJOruHZk
T topic9 6g0VIAsRQJJ3jIC
T topic0 HM0uWn3em3hp9nQ
U topic7 1
U topic9 1
T topic8 6Wsy3sg487WGKJV
T topic0 x6grqZtvNvKTPc5
U topic0 2
S topic9 0
S topic5 1
S topic0 2
S topic5 4
T topic1 J4o05V5JhyIh7Rq
S topic0 1
T topic7 tEP3IGGSedsHt3m
T topic9 0gsO3d59bGlBnuV
U topic2 3
S topic6 1
T topic4 Oo1y0ZxNFsSX9qi
T topic9 TDa6F6syXdf68aI
S topic0 0
T topic9 AxynjHPmFHMjFbp
U topic3 2
T topic0 cuJq8WyPVfjx34G
T topic9 CFKRxqPO8M16vDB
T topic5 nVKuy99YM6ybIcc
U topic4 4
S topic2 3
S topic1 4
S topic2 0
U topic6 0
T topic5 xTQNLEMj070Eucv
S topic4 3
S topic4 4
T topic1 1C9h6tLyrjU0thO
U topic3 0
S topic5 1
T topic0 HyisYSc1he0rZY2
S topic5 2
T topic4 gKtcIdGnoNopbvn
U topic0 2
S topic1 1
U topic1 1
U topic2 3
U topic9 2
T topic8 jODOgQXz9cHHGDK
S topic0 3
U topic6 4
S topic5 0
S topic2 4
S topic2 4
U topic3 2
S topic5 3
S topic8 2
U topic0 0
S topic2 0
S topic2 3
U topic8 2
S topic8 0
T topic4 r2O0EcBXE2jJaDq
U topic0 2
U topic3 4
S topic9 4
S topic0 1
U topic6 0
U topic4 2
U topic2 0
S topic1 3
U topic7 2
T topic8 E2Z5SmIAErToeGQ
S topic7 4
S topic0 2
S topic1 2
U topic7 4
U topic0 1
T topic7 8QM0ODfPvNBLUqr
S topic8 1
S topic6 2
U topic1 1
T topic0 4qZp76Yt6R1j66N
S topic4 1
U topic2 1
S topic3 2
T topic2 5KVMcserQYB3mUR
T topic4 2vPljPdxExVnKE8
S topic8 1
U topic0 3
S topic1 3
S topic4 3
U topic0 0
T topic3 uqoPvkq7P3MOQ7h
U topic9 2
S topic8 1
U topic6 0
T topic8 YiGvatBjdIHFWkS
U topic6 0
T topic6 pCdZAEtcBaagvvJ
S topic5 0
S topic0 1
T topic5 XsH8R2hg8tze7Gz
T topic5 KQ4gj0hdZFG2TNX
S topic9 3
U topic8 2
U topic0 3
U topic9 1
S topic3 3
U topic6 3
T topic5 ndCuxzXS5y0jmWm
U topic6 0
S topic0 3
U topic5 3
T topic5 7x1NMZoRdpDN0eg
U topic1 1
T topic7 hj7T8hq1Y6n1Or3
U topic3 3
U topic2 2
S topic7 1
S topic3 1
T topic9 Y4BSPZZeeeRLGG7
T topic9 uWx4mrJeIcEja3z
T topic3 B3eNq21GaxiJlpj
U topic4 4
T topic2 2SdYU6uj8EjsHfg
U topic2 2
T topic8 DTiP1avn5XPkUMH
U topic8 1
T topic3 km2F186hJCcgfjj
U topic1 1
T topic1 vezuMcYdnMDbJ8i
T topic0 R7U4l7KraaknQhk
S topic2 2
U topic0 1
U topic6 0
U topic5 0
U topic6 2
T topic1 n5Eo5QdM80BBams
S topic9 4
U topic9 0
U topic4 3
U topic9 4
U topic7 4
U topic5 0
T topic4 LcPOzjrSvUE0jV6
U topic8 4
T topic7 q3nQqjz4cNtqkPg
T topic0 VeZsDLPWcCax3S4
U topic9 4
S topic1 1
T topic3 BBiTy5qgqfo8rmg
U topic2 1
T topic4 zLNLl4c2viJrWGG
T topic3 hvE56JcvKgxKtAU
S topic5 4
U topic4 3
S topic8 3
U topic5 2
S topic8 4
S topic2 1
S topic9 3
S topic4 0
S topic8 3
U topic8 2
S topic3 3
S topic5 0
U topic7 1
U topic1 0
T topic8 YFFnzFdmE7iiksh
S topic9 0
U topic4 3
T topic5 yuPpsnsyyH83xYp
S topic2 3
U topic7 1
S topic9 1
U topic0 0
T topic1 hjkIEv2SbtBylm0
U topic8 3
U topic1 2
S topic7 3
T topic4 z2XneRZuCnOgeuJ
T topic5 jOE5f9Tk0uz5VXo
U topic8 2
U topic9 3
T topic8 SPvSkDu6nXIzB59
S topic8 4
S topic3 3
U topic2 2
U topic2 3
U topic5 2
U topic1 0
S topic7 4
S topic2 3
U topic4 0
U topic6 1
U topic3 2
T topic8 e1hGfiqa51bIhiE
S topic1 0
U topic0 0
S topic1 4
S topic1 2
U topic6 0
U topic0 0
U topic7 4
S topic9 4
S topic6 2
S topic4 4
T topic6 7YPSjS4ErG1vA9o
T topic0 Pdv93mavKqQnz9Z
T topic9 03qNBiJs0UrABYu
S topic2 0
S topic2 2
U topic6 3
T topic3 FyL3id3Ns1RsHCO
S topic9 1
T topic5 igd8BGaz8zcgmRC
U topic0 2
T topic1 kpYcNTGqWN0b6ie
U topic1 0
T topic2 r5nmRiulr5AWZiR
S topic2 3
U topic8 1
U topic0 3
U topic7 3
S topic3 2
T topic8 u3cfBKK4o2QYo7x
T topic2 ixm5eVFalPV28Ue
T topic1 9KmhlE7mTRPIJvl
S topic6 0
T topic5 AjgQa9I8tRdOums
T topic9 cSoSzjFJSVKsXT9
T topic3 u96jLvTY31FgBuC
T topic3 cC6Fm0Y8fmXX5nS
U topic7 4
S topic0 1
U topic7 2
U topic0 3
S topic0 0
T topic8 C9drrVN9JbOZPTk
U topic4 1
T topic6 vGk7Qhz44uMsn7g
S topic9 1
T topic2 Tkxt2efaC6oqFNN
T topic3 AR9C5lhvoFWLcDA
S topic5 2
U topic1 1
T topic0 JIUA63daS6Q2KDT
U topic5 1
U topic1 0
T topic5 rPOxVMk1MRaLB9x
S topic3 3
T topic8 72CMVx6B96CjOYs
T topic0 qJJ50le4T5iffIz
S topic9 4
T topic7 81RobHp7nBRL3Rc
T topic4 me5bBkjSpj2MT7E
U topic6 1
T topic5 tFaAXtNUjQTyCTN
U topic1 2
U topic0 3
U topic3 2
S topic3 1
T topic0 jRCZh49VnnVRhVg
T topic2 EC6ZzkfTkCRKRkX
S topic0 1
S topic3 0
S topic6 1
T topic1 pfaWGtxfGZDiQrA
T topic6 zKlJomJxeE4yyqk
T topic4 XQ2Vmd36D8VVBOG
S topic6 1
U topic0 2
U topic0 4
S topic5 4
S topic2 2
S topic2 1
S topic2 1
U topic1 4
U topic2 2
S topic5 2
T topic1 orIkL21KyGbcIYy
U topic5 0
T topic8 uXr9YIaKSXorN7E
U topic2 4
U topic6 3
U topic7 3
S topic1 2
U topic8 1
S topic5 2
S topic4 4
S topic1 4
S topic4 0
S topic5 0
T topic5 TgSKubyzF47VyyN
U topic1 4
S topic0 0
S topic6 4
T topic1 275T5kXBiHs6Ji4
T topic0 TOn1DireL3soFHC
U topic8 2
T topic5 OefxN6QoyxgpQcM
T topic6 kzH1pvL652kSQ3I
S topic9 4
U topic4 2
U topic3 0
U topic8 4
S topic8 1
T topic7 zjPak1NU33EFTTQ
U topic8 3
S topic1 2
S topic3 1
U topic4 4
U topic2 4
U topic6 1
U topic2 0
U topic9 0
S topic8 4
S topic8 4
U topic1 2
S topic6 1
S topic9 2
U topic6 3
U topic5 2
T topic1 hymqEu94UBIZTlr
S topic9 1
T topic8 iyltb4859DlfeDi
T topic1 d6Ny2KbpCVCbzHN
T topic5 8m6Sy8q765tDVOq
U topic1 2
T topic1 hTAFVVVl3MVhT73
U topic3 1
T topic3 AmYCbLW9sFE0MBS
