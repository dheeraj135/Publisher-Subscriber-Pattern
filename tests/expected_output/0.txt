5ptTXlASQRuu00l
23eKqe2877Cykgh
6tMET3Ua54gup3k
eHxcLalOvIjIItS
CykXu0yZvrnfihC
voAuMs8naO8EjAO
zFiR95FMsxnuasK
6e7C117WX7832bX
UqGjat6mjsneCs9
HSIXo5kuiH7npDv
eaHHyP22FPJ1UOm
Cgiy9CDqLfyxSFc
1T5dDlrbyWdBmm0
h4XLmb5ET7ohDPz
AGIRVByMoV2etdd
1hnvsO2GEvePzAy
30M4Yleq8I7RVBT
